### PR TITLE
Mat-9795 v1 displaying admin user layout

### PR DESCRIPTION
**Pull request by sb-prateekkeerthi**
## MADiE PR

Jira Ticket: [MAT-9795](https://jira.cms.gov/browse/MAT-9795)
(Optional) Related Tickets:

### Summary
added header for profiles

Screenshot 2026-05-26 at 12 23 06 PM

Screenshot 2026-05-26 at 12 05 25 PM

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included
* [ ] No extemporaneous files are included (i.e Complied files or testing results)
* [ ] This PR is into the **correct branch**.
* [ ] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [ ] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package)
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: src/components/PageHeader/PageHeader.tsx [hunk 1]
```diff
@@ -64,6 +102,17 @@ const PageHeader = () => {
     };
   }, []);
 
+  const [userInfo, setUserInfo] = useState<UserDetails | null>(
```
Comment (RohitKandimalla) [inline src/components/PageHeader/PageHeader.tsx]: How can we differentiate the logged in User information vs Admin selected user information. I feel we can have a precise variable names to avoid confusion.

File: src/components/PageHeader/PageHeader.tsx [hunk 2]
```diff
@@ -417,15 +466,63 @@ const PageHeader = () => {
...
+              </p>
             </div>
           </div>
-        </div>
-      )}
+        ) : (
+          <div className="admin">
+            <div>
+              <div className="left-col">
+                <h1> Administration </h1>
```
Comment (RohitKandimalla) [inline src/components/PageHeader/PageHeader.tsx:520]: Not sure if this means anything.

File: src/components/PageHeader/PageHeader.tsx (hunk 1 above)
Comment (sb-prateekkeerthi) [reply src/components/PageHeader/PageHeader.tsx]: sure I changed the variables to 

const [selectedUserProfileInfo, setSelectedUserProfileInfo] = useState<UserDetails | null>(
    adminUserStore.state
  );

File: src/components/PageHeader/PageHeader.tsx (hunk 2 above)
Comment (sb-prateekkeerthi) [reply src/components/PageHeader/PageHeader.tsx:520]: this is for the main page, I have not changed anything. I have attached the screenshots above

Comment (RohitKandimalla) [reply src/components/PageHeader/PageHeader.tsx:520]: Got it, The left margin of Administration is not aligning

**Review by RohitKandimalla — APPROVED** [review]
